views/templates/admin/configuration.tpl

```diff
@@ -22,3 +22,10 @@
 <div id="app"></div>
 
 <script src="{$pathApp|escape:'htmlall':'UTF-8'}"></script>
+
+<style>
+  /** Hide native multistore module activation panel */
+  #content div.bootstrap.panel {
```

Comment:
This seems to be a generic selector, this could have some side effects if other modules are displayed on this page?

Reply:
It's the PrestaShop Checkout configuration page so no other modules...
PrestaShop Checkout component are in #app and not affected by this.

Reply:
This is what you consider by default on a shop, but PrestaShop provides a hook which could allow modules to add content at the end of the page (you have heard of `displayAdminEndContent` with ps_mbo :slightly_smiling_face:).

If someone decided to add content and reusing the generic classes of Bootstrap, there is a risk that you alter other modules content. This solution is not as clean a yours, but it prevents side effects:

```suggestion
  $('input[name="activateModule"]', '#content').parent('.panel')
```

Reply:
I did that due to PrestaShop Core bug not handle correctly nobootstrap class.
We don't want any other modules to add content on this page with unattended display.
If you add content reusing the generic classes of Bootstrap it will not work well due to nobootstrap, so it's attended to hide them.